linode/resource_linode_instance_test.go

```diff
@@ -1673,6 +1673,18 @@ resource "linode_instance" "foobar" {
 }`, instance, pubkey)
 }
 
+func testAccCheckLinodeInstanceWithBootImage(identifier, instance string, image string) string {
```

Comment:
why bother accepting image? just assume the one version you have everywhere, and rename the fun?